### PR TITLE
scope handler optimization

### PR DESCRIPTION
**Pull request by kumavis**
this dramatically improved containment performance on my Dell Optiplex GX270

### CODE REVIEW

File: packages/ses/src/scope-handler.js [hunk 1]
```diff
@@ -65,6 +65,10 @@ export const createScopeHandler = (
         return undefined;
       }
 
+      if (prop === 'mellon') {
```
Comment (Jack-Works) [inline packages/ses/src/scope-handler.js:68]: What is `mellon`?

Comment (kumavis) [reply packages/ses/src/scope-handler.js:68]: https://www.youtube.com/watch?v=de19gIYgk-o

Comment (kriskowal) [reply packages/ses/src/scope-handler.js:68]: This was clearly a reference aimed directly at my hobbies.

**Comment by kriskowal** [comment]
Tempting, but no.

**Pull request closed without merging**